### DISCUSSION TITLE
hivesql error help me

### DISCUSSION THREAD

**Discussion by ColaSBear** [apache/dolphinscheduler#11191] [General]
dolphinscheduler:3.0.0
hive:3.1.3
cdp:7.1.7
hive engine:tez

execute hivesql fail but log no error
![356eee64108b8062c235750e7aeb375](https://user-images.githubusercontent.com/105793489/181511306-fb504c48-0736-4bf7-98ae-e2b9eac045c7.jpg)
![bd0a6050282ca006f6e5b673b440d35](https://user-images.githubusercontent.com/105793489/181511315-8938de19-7bf0-46da-a268-15760e6bed36.jpg)
Is there any special configuration？

**Comment by hstdream** [comment]
The error is that UDF resources are not found. Can you provide the following execution steps?

**Comment by Tandoy** [comment]
This is not a cause by error, just info. Maybe the error is reported in other places.
